src/lib/ComponentLoader.coffee

```diff
@@ -112,7 +112,7 @@ class ComponentLoader extends EventEmitter
 
   isGraph: (cPath) ->
     # Live graph instance
-    return true if typeof cPath is 'object' and cPath instanceof nofloGraph.Graph
+    return true if typeof cPath is 'object' and cPath instanceof fbpGraph.Graph
```

Comment:
... assuming that the code uses the exact same NoFlo version? But I guess that may have been a requirement before also.